src/ui/public/chrome/directives/app_switcher_link/__tests__/app-switcher-link.js

```diff
@@ -207,5 +208,15 @@ describe('appSwitcherLink directive', () => {
         expect(link.attr('title')).to.be(attrs.appSwitcherLinkTitle);
       });
     });
+
+    describe('appSwitcherLinkClasses attribute', () => {
+      it('is applied to the link', () => {
+        const attrs = {
+          appSwitcherLinkClasses: 'new classes',
+        };
+        const element = create(attrs);
+        expect(element.attr('class')).to.contain(attrs.appSwitcherLinkClasses);
```

Comment:
I've never been a big fan of tests that use the same data as input and as what they check. What if `create(attrs)` didn't work so `class` is wrong, but because you have a typo in `attrs.appSwitcherLinkClassses` (aka it's `undefined`) it doesn't fail? I much prefer actually typing it out.